_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -72,184 +68,143 @@ For value type `URI`, this rule does not require that the destination URI exists
 
 #### Passed Example 1
 
-`aria-required` property with valid true/false value
+The `aria-label` [attribute value][] of `Family name` is allowed for the `string` [value type][].
 
 ```html
-<div role="textbox" aria-required="true" aria-label="A required textbox"></div>
+<div role="textbox" aria-label="Family name"></div>
 ```
 
 #### Passed Example 2
 
-`aria-expanded` state with valid true/false/undefined value
+The `aria-required` [attribute value][] of `true` is allowed for the `true/false` [value type][].
 
 ```html
-<div role="button" aria-expanded="undefined">A button</div>
+<div role="textbox" aria-required="true" aria-label="Family name"></div>
 ```
 
 #### Passed Example 3
 
-`aria-pressed` state with valid tristate value
+The `aria-expanded` [attribute value][] of `undefined` is allowed for the `true/false/undefined` [value type][].
 
 ```html
-<div role="button" aria-pressed="mixed">An other button</div>
+<div role="button" aria-expanded="undefined">A button</div>
 ```
 
 #### Passed Example 4
 
-`aria-errormessage` property with valid ID reference value
+The `aria-pressed` [attribute value][] of `mixed` is allowed for the `tristate` [value type][].
 
 ```html
-<div role="textbox" aria-errormessage="my-error" aria-label="A textbox"></div>
+<div role="button" aria-pressed="mixed">Partially pressed button</div>
 ```
 
 #### Passed Example 5
 
-`aria-owns` property with valid ID reference list value
+The `aria-errormessage` [attribute value][] is an `ID Reference` [value type][]. Because `aria-errormessage` is not a required property for `textbox`, an element with a matching ID is not required.
```

Comment:
```suggestion
The `aria-errormessage` [attribute value][] is an `ID Reference` [value type][]. The presence of an element with a matching ID is not required by this rule.
```
The current version of the rule doesn't mention the fact that the property is required or not, so it shouldn't really be part of the description.

Reply:
I agree with @Jym77 in that this feels like we're diluting things but I wouldn't block this either.